docs/howto/kubernetes/gardener/hibernate-shoot-cluster.md

```diff
@@ -3,69 +3,58 @@ description: How to hibernate a Gardener-based Kubernetes cluster
-much, if at all. To save on costs, you can put the whole cluster in
-hibernation. If you do, then from that time on (and until you wake the
-cluster again), you will be paying *less* for the cluster.
+There will be times when you won't be using your {{k8s_management_service}}-based cluster much, if at all.
+To save on costs, you can put the whole cluster in hibernation.
+If you do, then from that time on (and until you wake the cluster again), you will be paying *less* for the cluster.
 
 ## Prerequisites
 
-We assume you have already used {{brand_container_orchestration}} to spin up a
-Kubernetes cluster, which is now humming away. If you've never done
-this before, please feel free to [follow this
-guide](create-shoot-cluster.md).
+We assume you have already used {{brand_container_orchestration}} to spin up a Kubernetes cluster, which is now humming away.
+If you've never done this before, please feel free to [follow this guide](create-shoot-cluster.md).
 
 ## Hibernating a cluster
 
-Fire up your favorite web browser and navigate to
-<https://{{gui_domain}}>. Make sure the vertical pane at the left-hand
-side of the page is in full view, then choose
-*Containers → [{{k8s_management_service}}](https://{{gui_domain}}/containers/gardener)*.
-In the main pane, you will see your {{k8s_management_service}} cluster.  Click anywhere
-on the corresponding row for a detailed view of the various cluster
-characteristics. Go to the *Status* tab and check if hibernation is
-possible.
+Fire up your favorite web browser and navigate to <https://{{gui_domain}}>.
+Make sure the vertical pane at the left-hand side of the page is in full view, then choose *Containers → [{{k8s_management_service}}](https://{{gui_domain}}/containers/gardener)*.
+In the main pane, you will see your {{k8s_management_service}} cluster.
+Click anywhere on its row for a detailed view of the various cluster characteristics.
+Bring up the *Status* tab and, in the *Constraints* section, check whether hibernation is possible.
+In the example below we have a shoot cluster based on Kubernetes 1.27, the latest supported at the time of this writing, and hibernation is indeed possible.
 
 ![{{k8s_management_service}} cluster status](assets/garhiber-01.png)
 
-If hibernation is indeed possible, this will be reflected in the
-*Constraints* section. To go ahead and actually hibernate the cluster,
-click the orange three-dot icon at the right-hand side of the cluster
-row, and from the pop-up menu that appears, select *Hibernate Cluster*.
+To go ahead and actually hibernate the cluster, click the orange :material-dots-horizontal-circle: icon at the right-hand side of the cluster row.
+From the pop-up menu that appears, select *Hibernate Cluster*.
 
 ![Hibernate cluster](assets/garhiber-02.png)
 
-A big pop-up window will appear, asking you to confirm the action.
+A big pop-up window labeled *About to hibernate a gardener shoot* appears, asking you to confirm the action.
```

Comment:
It strikes me as extremely odd that we're using all-lowercase "gardener" here, but I guess that's what the GUI does so we'll stick to it.

Reply:
It's the GUI, yes :/